composer.json

```diff
@@ -15,7 +15,8 @@
         "php": "^5.5.9|^7.0",
         "symfony/framework-bundle": "^2.8|^3.0",
         "sonata-project/seo-bundle": "^1.1.3|^2.0",
-        "doctrine/collections": "^1.0"
+        "doctrine/collections": "^1.0",
+        "phpcr/phpcr-utils": "^1.2.9"
```

Comment:
this has nothing to do with short array notation or not. i really don't get how 7.1 could get the error we see, it makes no sense if itreally installed 1.2.9 - or is the error in some other place this time?